mobile/src/navigation/app/components/drawer-content/drawer-content.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { TouchableOpacity, ScrollView } from 'react-native';
+import {
+  DrawerContentScrollView,
+  DrawerItemList,
+} from '@react-navigation/drawer';
+import {
+  DrawerDescriptorMap,
+  DrawerNavigationHelpers,
+} from '@react-navigation/drawer/lib/typescript/src/types';
+import { DrawerNavigationState, ParamListBase } from '@react-navigation/native';
+import { View, Text } from '~/components/common/common';
+import { styles } from './styles';
+
+const CustomDrawerContent = (
+  props: JSX.IntrinsicAttributes & {
+    state: DrawerNavigationState<ParamListBase>;
+    navigation: DrawerNavigationHelpers;
+    descriptors: DrawerDescriptorMap;
+  },
+): JSX.Element => {
```

Comment:
There is a library type for this
```suggestion
const CustomDrawerContent: FC<DrawerContentComponentProps> = (props) => {
```